src/main/java/org/cyclonedx/parsers/JsonParser.java

```diff
@@ -62,7 +63,9 @@ public Bom parse(final File file) throws ParseException {
      */
     public Bom parse(final byte[] bomBytes) throws ParseException {
         try {
-            return JSON.parseObject(bomBytes, Bom.class, Feature.SupportNonPublicField);
+            GsonBuilder builder = new GsonBuilder();
+            Gson gson = builder.create();
```

Comment:
Q: why is the use of builder here? (while in the other occurrences  just using `new Gson()`)